docs/components/docgen.js

```diff
@@ -50,6 +50,37 @@ export default async function docgen({
   return parsed;
 }
 
+export async function multipledocgen({
+  componentName,
+  alternativeSubdirectory = '',
+}: {|
+  componentName: Array<string> | string,
+  alternativeSubdirectory?: string,
+|}): Promise<Array<DocGen>> {
+  const parsedPropTables = await Promise.all(
+    (Array.isArray(componentName) ? componentName : [componentName]).map(async (cmp) => {
+      const filePath = path.join(
```

Comment:
Instead of repeating this logic, can any parts of this be extracted into shared helper functions?